app/src/main/res/layout/dialog_help.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+<LinearLayout 
```

Comment:
Please reformat the file so that the indentation is correct

Reply:
OK, I would like to fix this later. :)